spec/comment_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Comments, type: :model do
```

Comment:
Please write a test for `#comment_counter` in the Comment model as all custom methods in the model should be tested.